src/cross_toml.rs

```diff
@@ -120,12 +188,16 @@ impl CrossToml {
         self.targets.get(target)
     }
 
-    fn get_string(
-        &self,
+    fn get_string<'a>(
+        &'a self,
         target: &Target,
-        get: impl Fn(&CrossTargetConfig) -> &Option<String>,
+        get_build: impl Fn(&'a CrossBuildConfig) -> Option<&'a String>,
+        get_target: impl Fn(&'a CrossTargetConfig) -> Option<&'a String>,
```

Comment:
I wonder if having `Self::get_target` and `get_target` is going to be confusing. Should be fine, just wondering if there's a better naming scheme. Maybe `get_for_target`?

Reply:
can also remove the `get_` prefix, I think I prefer that